<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>importarincidencias</class>
 <widget class="QDialog" name="importarincidencias">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lblTipoArchivo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblNombreArchivo">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>41</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>115</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Tipo de archivo...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>230</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
   <property name="icon">
    <iconset resource="imagenes/imagenes.qrc">
     <normaloff>:/cerrar.jpeg</normaloff>:/cerrar.jpeg</iconset>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnImportar">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>190</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Importar</string>
   </property>
   <property name="icon">
    <iconset resource="imagenes/imagenes.qrc">
     <normaloff>:/importardatos.png</normaloff>:/importardatos.png</iconset>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>561</width>
     <height>67</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,1">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>Periodo de datos</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Desde:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblDesde">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Hasta:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblHasta">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnAbrirArchivo">
   <property name="geometry">
    <rect>
     <x>559</x>
     <y>41</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>311</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Archivo de Incidencias a importar ...</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="imagenes/imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
